<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardForm</class>
 <widget class="QWidget" name="KeyboardForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
background-color: rgb(94, 92, 100);
border: 3px solid #FFFFFF;
border-radius: 20px;
}

QPushButton{
color: rgb(255, 255, 255);
font-size: 20pt;
border-image: url(:/png/biankuang/按钮7.png);
}
QPushButton::pressed{
color: #0663A1;
	border-image: url(:/png/biankuang/按钮-边框.svg);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnMinus">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
padding-top: 10px;
padding-bottom: 10px;
image: url(:/png/biankuang/backspace (1).svg);
}
QPushButton::pressed{
padding-top: 10px;
padding-bottom: 10px;
image: url(:/png/biankuang/backspace (5).svg);
}</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn7">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn8">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn9">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">9</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn5">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn6">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">6</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn1">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn0">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDot">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEnter">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	image: url(:/png/biankuang/回车.svg);
}
QPushButton::pressed{
	image: url(:/png/biankuang/回车 (5).svg);
}</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
